services/app-api/prisma/schema.prisma

```diff
@@ -291,12 +292,12 @@ model UserAudit {
 }
 
 model Question {
-  id            String                 @id @default(uuid())
-  createdAt     DateTime               @default(now())
-  updatedAt     DateTime               @default(now()) @updatedAt
-  pkgID         String
-  pkg           HealthPlanPackageTable @relation(fields: [pkgID], references: [id])
```

Comment:
With these changes to `Question` and `HealthPlanPackageTable`, would we want to make a migration or drop the Q&A tables? I think this only applies to `Val` since it is the only environment that has existing Q&A records.

Reply:
Take a closer look at that migration! All the Questions that were associated with HPP should now be associated with Contracts